lib/libutee/include/tee_internal_api_extensions.h

```diff
@@ -34,6 +34,17 @@ TEE_Result TEE_CacheClean(char *buf, size_t len);
 TEE_Result TEE_CacheFlush(char *buf, size_t len);
 TEE_Result TEE_CacheInvalidate(char *buf, size_t len);
 
+/*
+ * Send an OCALL to the Client Application.
+ *
+ * The semantics are identical to TEEC_InvokeCommand, except that memory
+ * references with size zero are not supported, in the opposite direction.
+ */
+TEE_Result TEE_InvokeCACommand(uint32_t cancellationRequestTimeout,
```

Comment:
Sorry nitpicking on labels, I think it would be clearer if this is renamed `TEE_InvokeOCall()` or `TEE_InvokeOCallCommand()`

Reply:
Nothing to be sorry about, I welcome feedback, it makes me think about my silly ideas in different ways.

I'm not opposed to the change. The reason I chose `TEE_InvokeCACommand` is because it neatly mirrors `TEE_InvokeTACommand`, but `TEE_InvokeOcall` would also work.

@jenswi-linaro, what do you say?

Reply:
I liked `TEE_InvokeCACommand()` too because it neatly mirrors `TEE_InvokeTACommand()`. I don't have a strong opinion on this.

Reply:
+1 for `TEE_InvokeCACommand()`.